diff --git a/configure.cmd b/configure.cmd
index 270c253881..0d1eaf3dce 100755
--- a/configure.cmd
+++ b/configure.cmd
@@ -39,7 +39,7 @@ set CMAKE_GENERATOR="Ninja"
 set CMAKE_GENERATOR_HOST=!CMAKE_GENERATOR!
 
 REM Detect presence of cmake
-cmd /c cmake --version 2>&1 | find "cmake version" > NUL || goto cmake_notfound
+cmd /c cmake --version 2>&1 | %windir%\system32\FIND.exe "cmake version" > NUL || goto cmake_notfound
 
 REM Detect build environment (MinGW, VS, WDK, ...)
 if defined ROS_ARCH (
@@ -50,16 +50,16 @@ if defined ROS_ARCH (
 
 ) else if defined VCINSTALLDIR (
     REM VS command prompt does not put this in environment vars
-    cl 2>&1 | find "x86" > NUL && set ARCH=i386
-    cl 2>&1 | find "x64" > NUL && set ARCH=amd64
-    cl 2>&1 | find "ARM" > NUL && set ARCH=arm
-    cl 2>&1 | find "15.00." > NUL && set VS_VERSION=9
-    cl 2>&1 | find "16.00." > NUL && set VS_VERSION=10
-    cl 2>&1 | find "17.00." > NUL && set VS_VERSION=11
-    cl 2>&1 | find "18.00." > NUL && set VS_VERSION=12
-    cl 2>&1 | find "19.00." > NUL && set VS_VERSION=14
-    cl 2>&1 | find "19.10." > NUL && set VS_VERSION=15
-    cl 2>&1 | find "19.11." > NUL && set VS_VERSION=15
+    cl 2>&1 | %windir%\system32\FIND.exe "x86" > NUL && set ARCH=i386
+    cl 2>&1 | %windir%\system32\FIND.exe "x64" > NUL && set ARCH=amd64
+    cl 2>&1 | %windir%\system32\FIND.exe "ARM" > NUL && set ARCH=arm
+    cl 2>&1 | %windir%\system32\FIND.exe "15.00." > NUL && set VS_VERSION=9
+    cl 2>&1 | %windir%\system32\FIND.exe "16.00." > NUL && set VS_VERSION=10
+    cl 2>&1 | %windir%\system32\FIND.exe "17.00." > NUL && set VS_VERSION=11
+    cl 2>&1 | %windir%\system32\FIND.exe "18.00." > NUL && set VS_VERSION=12
+    cl 2>&1 | %windir%\system32\FIND.exe "19.00." > NUL && set VS_VERSION=14
+    cl 2>&1 | %windir%\system32\FIND.exe "19.10." > NUL && set VS_VERSION=15
+    cl 2>&1 | %windir%\system32\FIND.exe "19.11." > NUL && set VS_VERSION=15
     if not defined VS_VERSION (
         echo Error: Visual Studio version too old or version detection failed.
         endlocal
